### ISSUE TITLE
shinyDirChoose() with getVolumes() only select C:/ in windows 7

### ISSUE THREAD

**Issue by olivier6088** [thomasp85/shinyFiles#40]
Hi,

Under Windows 7 the function shinyDirChoose() coupled with getVolumes() is unable to select other volume but C:/

![image](https://cloud.githubusercontent.com/assets/10126543/17206717/27a2c472-54c2-11e6-89bf-e0f254f9864f.png)

The problem comes from getVolumes().

A simple solution is to change the code in windows case by the following code:

```r
else if (osSystem == "Windows") {
        volumes <- system("wmic logicaldisk get name", intern = T)
        volumes <- sub(" *\\r$", "", volumes)
        keep <- !tolower(volumes) %in% c("name", "")
        volumes <- volumes[keep]
        names(volumes) <- volumes
      }
``` 

The problem came from the parenthesis `(`, `)` on the volumes' names.

By the way there is no need to launch the system command twice !

Here is the result after correction:

![image](https://cloud.githubusercontent.com/assets/10126543/17206815/9badac60-54c2-11e6-9ae3-faf9187c9248.png)

**Comment by thomasp85** [comment]
Thanks for this. The fix will be implemented fastest if you make a PR, but I'll eventually get to it anyway

**Comment by olivier6088** [comment]
Good to know, it's my first issue on somebody else repository !
Never done a PR before...

**Comment by thomasp85** [comment]
This is a good chance to try it then - I'll be happy to walk you through it once I'm back from vacation next week. You already have the fix so it's mostly about how to deal with the whole PR thingy...

**Comment by thomasp85** [comment]
Fixed in 8202f00
